README.md

```diff
@@ -114,41 +111,124 @@ to exist but could be an empty file.
 
 ### Apache 2.0 - 2.2
 
-If you wish to configure your FastCGI application to work with the apache web server, you can use the apache FastCGI module to process manage your application.
+If you wish to configure your FastCGI application to work with the apache web server, 
+you can use the apache FastCGI module to process manage your application.
 
-This can be done by creating a FCGI script that launches your application and inserting a FastCgiServer directive into your virtual host configuration.
+This can be done by creating a FCGI script that launches your application and inserting 
+a FastCgiServer directive into your virtual host configuration.
 
 Here is an example `script.fcgi`:
 
 ```sh
 #!/bin/bash
+
+# Run the server
 php /path/to/application.php run
 ```
 
-Or with Symfony:
+See other commands to run the server [here](#running-the-server).
 
-```sh
-#!/bin/bash
-php /path/to/bin/console speedfony:run --env=prod
-```
 
-In your configuration, you can use the [FastCgiServer](https://web.archive.org/web/20150913190020/http://www.fastcgi.com/mod_fastcgi/docs/mod_fastcgi.html#FastCgiServer) directive to inform Apache of your application.
+In your configuration, you can use the [FastCgiServer](https://web.archive.org/web/20150913190020/http://www.fastcgi.com/mod_fastcgi/docs/mod_fastcgi.html#FastCgiServer) 
+directive to inform Apache of your application.
 
 ```
 FastCgiServer /path/to/script.fcgi
 ```
 
-By default, the daemon will listen on FCGI_LISTENSOCK_FILENO, but it can also be configured to listen on a TCP address. For example:
+## Running the server
 
+Depending on your setup, you will have different ways of running the server. In
+a normal PHP application where you have created your own `fastCGI_app.php` ((see how)[#usage]),
+you may start the server simply by: 
 
-```sh
-#!/bin/bash
-php /path/to/application.php run --port=5000 --host=localhost
+```bash
+php /path/to/fastCGI_app.php run
 ```
 
-Or with Symfony:
+In a Symfony application where you have registered `DaemonRunCommand` as a service, 
+you may just run: 
 
-```sh
-#!/bin/bash
-php /path/to/bin/console speedfony:run --env=prod --port=5000 --host=localhost
+```bash
+# If installed with Symfony Flex
+./bin/console speedfony:run fastcgi-daemon:run
```

Comment:
Would this just be:

`./bin/console fastcgi-daemon:run`?

Reply:
Yeah, Typo